rest/src/main/java/org/jboss/pnc/rest/provider/BuildRecordProvider.java

```diff
@@ -229,6 +229,15 @@ public String getBuildRecordLog(Integer id) {
             return null;
     }
 
+    public String getBuildRecordRepourLog(Integer id) {
+        BuildRecord buildRecord = ((BuildRecordRepository) repository).findByIdFetchAllProperties(id);
```

Comment:
When we do a request to build/repour log also the second log is obtained from the database it affects the performance of these requests as that are LOBs - NCL-2989 created